package.json

```diff
@@ -43,6 +43,10 @@
     "aegir": "^12.0.8",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-webworker": "^1.3.0",
```

Comment:
This gets installed with `aegir`. Did it not work for you?

Reply:
@diasdavid It was working on my machine, but the CI builds were failing (especially, Jenkins) as shown in my checkin history. I added these to make the CI builds work. I'll revert it back.